index.html

```diff
@@ -0,0 +1,331 @@
+<!DOCTYPE html>
```

Comment:
I'm not sure if this is in the scope of the task, but I think extracting some re-usable components (header, card, etc...) would be helpful here.

Reply:
@AhOmran done.